src/app/app.module.ts

```diff
@@ -21,7 +23,9 @@ import { DualSliderModule } from './shared/components/dual-slider/dual-slider.mo
     OntologyExplorationModule,
     InfoDialogModule,
     MatIconModule,
-    DualSliderModule
+    DualSliderModule,
+    ImageViewerPopoverModule,
```

Comment:
Image viewer will have a lot of stuff in it, probably best to have an image viewer module that imports those and exports them, so down below its just ImageViewerModule.

Reply:
Okay everything is in the ImageViewerModule now